src/backend/marsha/core/permissions.py

```diff
@@ -35,15 +36,15 @@ def has_permission(self, request, view):
         user = request.user
         if (
             isinstance(user, TokenUser)
-            and LTI_ROLES[INSTRUCTOR] & set(user.token.payload.get("roles", []))
+            and LTI_ROLES[self.role] & set(user.token.payload.get("roles", []))
```

Comment:
since `role` is a class attribute, it may be safer and clearner to poke it on the class:
```suggestion
            and LTI_ROLES[self.__class__.role] & set(user.token.payload.get("roles", []))
```